react-native/training/src/app/screens/Home/screens/TodoList/layout.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { View, ScrollView } from 'react-native';
+import PropTypes from 'prop-types';
+
+import CustomTextInput from '../../../../components/CustomTextInput';
+
+import styles from './styles';
+import Item from './components/Item';
+import Footer from './components/Footer';
+
+function Layout({ addItem, selectItem, deleteItem, items, deleteItemSelected }) {
+  return (
+    <View style={styles.container}>
+      <CustomTextInput placeholder="Enter an item!" onSubmitEditing={addItem} style={styles.inputElement} />
```

Comment:
Use i18n here. Check examples in the proyect. A correct translation is like `strings.THIS_IS_A_TRANSLATION();`